Description: fix a regression that affects methods with %MethodCode and __len__ annotation
Origin: upstream, http://riverbankcomputing.co.uk/hg/sip/rev/765b6874363f
Last-Update: 2015-03-16

--- a/sipgen/metasrc/parser.y
+++ b/sipgen/metasrc/parser.y
@@ -7172,7 +7172,7 @@
         len->common = findFunction(pt, mod, c_scope, mt_scope, len->cppname,
                 TRUE, 0, FALSE);
 
-        if (od->methodcode == NULL)
+        if ((len->methodcode = od->methodcode) == NULL)
         {
             char *buf = sipStrdup("            sipRes = (SIP_SSIZE_T)sipCpp->");
             codeBlock *code;
